java/rocksjni/env_flink.cc

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
```

Comment:
Is this ready for this pr?

Reply:
I have removed this file in the latest commit. The JNI support of FlinkEnv will be introduced in another pr.